integrations/extensions/docs/elasticsearch-install-and-setup/how_to_install_docker.md

```diff
@@ -0,0 +1,141 @@
+# Guide to installing Docker or Docker alternatives
+
+## What is Docker?
+
+Docker is a software platform that allows you to build, test, and deploy applications quickly. Docker packages software into standardized units called containers that have everything the software needs to run including libraries, system tools, code, and runtime. Using Docker, you can quickly deploy and scale applications into any environment and know your code will run.
+
+And Docker Compose is a tool for defining and running multi-container applications. It is the key to unlocking a streamlined and efficient development and deployment experience.
+
+You will see references to `docker` and `docker-compose` as you work through some of our guides and this document serves to guide anyone who needs a starting point to install that software or its alternatives.
+
+Useful links:
+
+1. [Docker Compose Overview](https://docs.docker.com/compose/)
+2. [Docker Overview](https://docs.docker.com/get-docker/) 
+
+## Option 1: Docker Desktop
+
+**IMPORTANT NOTE**: Docker Desktop is free for small businesses (fewer than 250 employees AND less than 10 dollar million in annual revenue), personal use, education, and non-commercial open source projects. Otherwise, it requires a paid subscription
+
+Docker provides a simple and easy to use desktop platform that comes integrated with the `docker` engine as well as `docker-compose`. It is easy to use and is maintained regularly by Docker. 
+
+If you have an existing subscription or if your enterprise is able to provide you with a subscription, you can follow instrutions to download and install ***Docker Desktop*** for the OS that you are working with, either [Mac](https://docs.docker.com/desktop/install/mac-install/), [Windows](https://docs.docker.com/desktop/install/windows-install/) or [Linux](https://docs.docker.com/desktop/install/linux-install/).
+
+Keep in mind that you would need to activate a subscription in case you are not eligible for the free version of Docker Desktop.
+
+## Option 2: Rancher Desktop
+
+**IMPORTANT NOTE**: Recommended for users who are new to Docker and prefer an easy one-click install of basic functionality
+
+### For Mac and Windows: 
+
+Go to the Rancher Desktop [homepage](https://rancherdesktop.io/) and download the appropriate installer for your OS , then proceed to install using your respective installer file which should help install the software automatically on your machine
+
```

Comment:
After I install Rancher Desktop on my Mac, I run the app and I get this screen:

<img width="401" alt="image" src="https://github.com/watson-developer-cloud/assistant-toolkit/assets/11746051/499e7ec6-af21-47f6-aac5-4b10993c5182">

I think it would be helpful to tell users to run this program and what options to select on this screen to get the instructions in our guides to work.